corehq/apps/userreports/models.py

```diff
@@ -151,9 +151,17 @@ def indicators(self):
                 }
             }
         }, self.named_filter_objects)
+        repeat_iteration_indicator = IndicatorFactory.from_spec({
+            "column_id": "repeat_iteration",
+            "type": "repeat_iteration",
+            "display_name": "base document iteration",
+            "datatype": "integer",
+            "is_nullable": False,
+            "is_primary_key": True,
+        }, self.named_filter_objects)
```

Comment:
I thought maybe it would be, but looks like it's actually easy to only add it to data sources with a base item expression. Fixed in f2be6c4.